_posts/2020-08-28-eslint-public-roadmap.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Announcing ESLint's public roadmap"
+tags:
+  - Sponsorships
+  - Donations
+  - Roadmap
+  - Planning
+---
+
+# Announcing ESLint's public roadmap
+
+When ESLint [first started accepting donations](https://eslint.org/blog/2019/02/funding-eslint-future), we listed several ways that the project intended to spend the money donated:
+
+* **Paying team members for maintenance and development.** We hired our first part-time maintainer, [Kai Cataldo](https://github.com/kaicataldo), and started paying TSC members on a per-hour basis up to $1,000 each month (the maximum we can afford at the moment).
+* **Rewarding contributions from the community.** We have given spot bonuses to people who have made significant contributions to ESLint.
+* **Improve documentation.** We are participating in the [2020 Google Season of Docs](https://eslint.org/blog/2020/08/eslint-google-season-of-docs-project-writer) to improve our documentation.
+* **Better communication and support.** Paying team members has allowed us to participate in the monthly TC-39 tooling meeting and allowed us to pay contributors to help answer questions both in GitHub and in Discord.
```

Comment:
The folks at Stack\&nbsp;Overflow have me double-checking punctuation in proper names :laughing:

```suggestion
* **Better communication and support.** Paying team members has allowed us to participate in the monthly TC39 tooling meeting and allowed us to pay contributors to help answer questions both in GitHub and in Discord.
```